en/patterns/dialogs.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Dialogs - Design System Pattern
+_description: The Dialogs Pattern Symbols are used to get user's attention and direct it toward important information and/or action they need to confirm and/or take.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+# Dialogs
+
+Use the Dialogs Pattern to alert or give the user relevant information for an error or a system state that requires urgent user input. Dialogs are usually triggered by a user's action and should be displayed over the main content. It's best when dialogs are used sparingly as they are in a way distracting to the user.
+
+<img class="responsive-img" src="../images/dialogs_demo.png" srcset="../images/dialogs_demo@2x.png 2x" />
```

Comment:
Replace this image with an image of the first dialog because that is what you reference with the layers screenshot below.

Reply:
Done